client/jetpack-cloud/sections/partner-portal/license-lightbox/index.tsx

```diff
@@ -24,11 +24,15 @@ export type LicenseLightBoxProps = {
 	extraAsideContent?: JSX.Element;
 	className?: string;
 	quantity?: number;
+	isCTAExternalLink?: boolean;
```

Comment:
Would it be better to check if `ctaHref` is defined and decide whether it's an external link or not, rather than creating a separate property?

Reply:
The reason I kept it like this is that we might have scenarios where the CTA button may have a link that is not external.

Reply:
Make sense 👍